lib/internal/errors.js

```diff
@@ -53,9 +52,20 @@ class AssertionError extends Error {
   }
 }
 
+// This is defined here instead of using the assert module to avoid a
+// circular dependency. The effect is largely the same.
+function assert(condition, message) {
+  if (!condition) {
+    throw new AssertionError({
+      message,
+      actual: false,
+      expected: true,
+      operator: '=='
```

Comment:
@evanlucas if I'm not wrong there only `assert.strictEqual()` is currently used in this file.